layers/osm.py

```diff
@@ -179,3 +179,41 @@ def roadHasBusLane(tags):
     # or any other users.
 
     return False
+
+def makeCrossings(
```

Comment:
If you've got the `black` formatter handy, could you run it here? If not, don't worry, I can get it later. Really need to use pyenv or poetry or something so we can do this properly